package.json

```diff
@@ -17,6 +17,7 @@
     "start:dev": "cross-env NODE_ENV=development ts-node -r dotenv/config src/server.ts",
     "start:dev:watch": "cross-env NODE_ENV=development ts-node-dev --clear --exit-child --watch src -r dotenv/config src/server.ts",
     "start:prod": "tsc && cross-env NODE_ENV=production node -r dotenv/config dist/src/server.js",
+    "start:free-ports": "npx cross-port-killer 3000",
```

Comment:
But that script doesn't start anything, no?
Naming is a bin confusing as it implies that the application will be started after this script.

Reply:
Removed `start` for clarity